activerecord/lib/active_record/readonly_attributes.rb

```diff
@@ -15,7 +15,8 @@ module ClassMethods
       # Attributes listed as readonly will be used to create a new record but update operations will
       # ignore these fields.
       #
-      # You can assign a new value to a readonly attribute, but it will be ignored when the record is updated.
+      # If `config.active_record.raise_on_assign_to_attr_readonly` is true, then if you assign a new
+      # value to a readonly attribute it will raise an error. This the default from Rails 7.1.
```

Comment:
I’m not sure we mention Rails defaults in rdoc, and as this will be released in Rails 7.1 we can probably just say it’s the default. Older versions won’t have this setting. 

```suggestion
      # value to a readonly attribute it will raise an error. This is the default.
```